torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -112,43 +128,82 @@ struct KinetoThreadLocalState : public ProfilerThreadLocalState {
       c10::Device device) override {
     if (config_.profile_memory && config_.state != ProfilerState::Disabled) {
       std::lock_guard<std::mutex> guard(state_mutex_);
+      auto start_time = getTimeUs();
+#ifdef USE_KINETO
```

Comment:
So now memory related event is added as a `libkineto::GenericTraceActivity` meaning this feature is available only if USE_KINETO